.github/workflows/ci.yml

```diff
@@ -116,11 +168,159 @@ jobs:
         with:
             command: test
             args: "--workspace \
-                   --package ark-test-curves \
-                   --all-features
-                   "
+                    --package ark-test-curves \
+                    --all-features
+                    "
         if: matrix.rust == 'nightly'
 
+  directories: # Job that list subdirectories
+    name: List directories for parallelizing tests
+    runs-on: ubuntu-latest
+    outputs:
+      dir: ${{ steps.set-dirs.outputs.dir }} # generate output name dir by using inner step output
+    steps:
+      - uses: actions/checkout@v4
+      - id: set-dirs # Give it an id to handle to get step outputs in the outputs key above
+        run: |
+          # shellcheck disable=SC2086
+          echo "dir=$(find . -mindepth 1 -maxdepth 1 -type d | jq -R -s -c 'split("\n")[:-1]')" >> $GITHUB_OUTPUT
```

Comment:
If anyone knows how to fix this without the ignore LMK

`.github/workflows/ci.yml:184:9: shellcheck reported issue in this script: SC2086:info:1:90: Double quote to prevent globbing and word splitting [shellcheck]`